Header header
string string_uri
---
uint8 error # 0 bei ok, != 0 sonst
